packages/2018-disaster-resilience/src/components/ViolentShakingAndGroundDeformation/index.js

```diff
@@ -1,25 +1,295 @@
 import React from 'react';
+import * as d3 from 'd3';
+import { CivicSandboxCard } from '@hackoregon/component-library';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+const wallOfText =
+  'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
+
+const dataURLs = [
+  'https://gist.githubusercontent.com/mendozaline/f78b076ce13a9fd484f6b8a004065a95/raw/ff8bd893ba1890a6f6c20265f720587f9595a9c4/pop.json',
+  'https://gist.githubusercontent.com/mendozaline/b3a75b40c9a60781b6adc77cebb9b400/raw/fa0aa13c75bfcc2fd92ccf1f3cc612af83d5d704/010-grocery.json',
+];
+
+class ViolentShakingAndGroundDeformation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      foundation1: null,
+      slide1: null,
+      error: null,
+    };
+  }
 
-export class ViolentShakingAndGroundDeformation extends React.Component {
   componentDidMount() {
-    // initialize data here
+    let cmp = this;
+    d3.queue()
+      .defer(d3.json, dataURLs[0])
+      .defer(d3.json, dataURLs[1])
+      .await((error, foundation1, slide1) => {
+        if (error) {
+          cmp.setState({ error: error });
+        }
+        cmp.setState({ foundation1, slide1 });
+      });
   }
 
   render() {
-    return (
-      <CivicStoryCard
-        title="Violent Shaking and Ground Deformation"
-        slug="violent-shaking-and-ground-deformation"
-      >
-        <Placeholder issue="217" />
-      </CivicStoryCard>
-    );
+    if (this.state.foundation1 === null) {
+      return null;
+    }
+    const { foundation1, slide1 } = this.state;
+    return dashboardComponent(foundation1, slide1);
   }
 }
 
-ViolentShakingAndGroundDeformation.displayName = 'ViolentShakingAndGroundDeformation';
+const dashboardComponent = (foundationData1, slideData1) => {
+  if (foundationData1 === null) {
+    return null;
+  }
+
+  const mapboxToken =
+    'pk.eyJ1IjoidGhlbWVuZG96YWxpbmUiLCJhIjoiY2o1aXdoem1vMWtpNDJ3bnpqaGF1bnlhNSJ9.sjTrNKLW9daDBIGvP3_W0w';
```

Comment:
I wonder if we should set the mapboxToken as an env var?

Reply:
It actually should be set in BaseMap by default, so should be able to just remove this.